pkg/occlient/occlient.go

```diff
@@ -545,15 +545,7 @@ func (c *Client) NewAppS2I(name string, builderImage string, gitUrl string, labe
 					},
 				},
 			},
-			Triggers: []buildv1.BuildTriggerPolicy{
-				{
-					Type: "ConfigChange",
-				},
-				{
-					Type:        "ImageChange",
-					ImageChange: &buildv1.ImageChangeTrigger{},
-				},
-			},
+			Triggers: []buildv1.BuildTriggerPolicy{},
```

Comment:
nit: we can simply drop the `Triggers:` field, we don't need it at all :man_shrugging:

Reply:
You are right.

I wanted to leave it here so everyone can quickly see that we are not setting any triggers.

Reply:
removed  it ;-)